clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/provider/agent/AppengineServerGroupCachingAgent.groovy

```diff
@@ -100,6 +100,9 @@ class AppengineServerGroupCachingAgent extends AbstractAppengineCachingAgent imp
     }
 
     def serverGroupName = data.serverGroupName.toString()
+    if (credentials.omitVersions != null && credentials.omitVersions.size() > 0 && credentials.omitVersions.contains(serverGroupName)) {
```

Comment:
replace with `shouldIgnoreVersion(serverGroupName)`

Reply:
Also, we generally prefer `.isEmpty` to `.size() > 0`